x-pack/test/functional/services/ml/test_resources.ts

```diff
@@ -128,6 +128,23 @@ export function MachineLearningTestResourcesProvider({ getService }: FtrProvider
       return createResponse.id;
     },
 
+    async createDashboardSavedObject(
```

Comment:
I think we have this functionality already in `createDashboard` (just missing the `override` option). Is there a reason to have a separate method?

Reply:
You're right. My apologies - didn't see the existing `createDashboard` previously. I ended up changing this to `createBulkSavedObjects` instead because it makes more sense in this context, and because it will give clearer error messages if there's any conflicts with the objects.